style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+#wrap {
+  width: 375px;
+  height: 638px;
+}
```

Comment:
- [ ] Dear @vikas924, Although your design is great, We are expected to make sure the mobile version is responsive until the desktop screen size. and as you can see your design is limited to `375px` screen size. 

![Screenshot 2023-02-16 172243](https://user-images.githubusercontent.com/69077061/219393149-2570581c-4b31-4a44-8eeb-4b7384ef6b9d.png)